src/universal/validation/templates.js

```diff
@@ -77,20 +77,19 @@ export const orgRejectionReason = (value) => value
 
 export const preferredName = (value) => value
   .trim()
-  .required('That\'s not much of a name, is it?')
-  .min(2, 'C\'mon, you call that a name?')
+  .required('That’s not much of a name, is it?')
+  .min(2, 'C’mon, you call that a name?')
   .max(100, 'I want your name, not your life story');
 
 export const task = (value) => value
   .trim()
-  .min(2, 'That doesn\'t seem like much of a task')
+  .min(2, 'That doesn’t seem like much of a task')
   .max(PROJECT_MAX_CHARS, 'Try shortening down the task name');
 
 export const teamName = (value) => value
   .trim()
-  .required('"The nameless wonder" is better than nothing')
-  .min(2, 'The "A Team" had a longer name than that')
-  .max(50, 'That isn\'t very memorable. Maybe shorten it up?');
+  .required('“The nameless wonder” is better than nothing')
```

Comment:
ok, I was intentional on updating with both opening and closing, but perhaps I messed up and/or was trumped by the auto-editing (I understand what you mean now)

I’ll go back and double check, thanks for catching that!

And yes, with any copy visible to humans, my preference is to use accurate punctuation.